src/quo2/components/list_items/saved_contact_address/view.cljs

```diff
@@ -0,0 +1,94 @@
+            account-props  (when (= accounts-count 1)
+                             (first accounts))
+            on-press-out   (fn []
+                             (let [new-state (if @active? :default :active)]
+                               (when @timer (js/clearTimeout @timer))
+                               (reset! timer nil)
+                               (reset! active? (= new-state :active))
+                               (reset! state new-state)
+                               (when on-press
+                                 (on-press))))]
+        [rn/pressable
+         {:style               (style/container
+                                {:state @state :customization-color customization-color})
+          :on-press-in         on-press-in
+          :on-press-out        on-press-out
+          :accessibility-label :container}
+         [rn/view {:style style/left-container}
+          [user-avatar/user-avatar (assoc contact-props :size :small)]
+          [rn/view {:style style/saved-contact-container}
+           [rn/view
+            {:style style/account-title-container}
+            [text/text
+             {:weight :semi-bold
+              :size   :paragraph-1}
+             (:full-name contact-props)]
+            [rn/view
+             {:style               style/contact-icon-container
+              :accessibility-label :contact-icon}
+             [icon/icon :i/contact
+              {:no-color true
+               :size     12
+               :color    (colors/theme-colors
+                          (colors/custom-color :blue 50)
+                          (colors/custom-color :blue 60)
+                          theme)}]]]
+           (if account-props
+             [rn/view
+              {:accessibility-label :account-container
+               :style               style/account-container}
+              [account-avatar/view
+               {:emoji (:emoji account-props)
+                :size  16}]
+              [text/text
+               {:size   :paragraph-2
+                :weight :monospace
+                :style  (style/account-name theme)}
+               (:name account-props)]
+              [rn/view {:style (style/dot-divider theme)}]
+              [text/text
+               {:size   :paragraph-2
+                :weight :monospace
+                :style  (style/account-address theme)}
+               (:address account-props)]]
+             [text/text
+              {:accessibility-label :accounts-count
+               :size                :paragraph-2
+               :style               (style/accounts-count theme)}
+              (i18n/label :t/accounts-count {:count accounts-count})])]]
+         [rn/view {:accessibility-label :check-icon}
+          [icon/icon :i/chevron-right
+           {:size  20
+            :color (colors/theme-colors colors/neutral-50
+                                        colors/neutral-40
+                                        theme)}]]]))))
```

Comment:
- I believe we will show the `chevron-right` icon if we have more than one account.
 - If the wrapper view is used only for accessibility, we can remove it and pass the `accessibility-label` to the icon.

Reply:
True! Nice catch, fixed it